src/go/rpk/pkg/cli/debug/bundle/bundle_linux.go

```diff
@@ -578,6 +579,37 @@ func saveCmdLine(ps *stepParams) step {
 	}
 }
 
+func saveRedpandaProc(ps *stepParams) step {
+	return func() error {
+		pid := getRedpandaPID(ps.fs)
+		if pid == 0 {
+			return errors.New("Unable to locate Redpanda PID")
```

Comment:
This means Redpanda is stopped, which might be normal in a crash scenario and it doesn't mean that is an error on the debug bundle collection, right?

Maybe:
```suggestion
			return errors.New("unable to locate Redpanda PID; is Redpanda process running? skipping /proc/<PID> collection")
```